src/components/tag-list/tag-list.css

```diff
@@ -0,0 +1,5 @@
+@import '../../styles/spacing.css';
+
+.tag-list > [role="row"] {
```

Comment:
Good choice of selector here. Nice way of enforcing the accessibility roles thru styles!